deploy/template.yaml

```diff
@@ -3128,36 +2970,6 @@ Resources:
         SSEType: KMS
         KMSMasterKeyId: !GetAtt DynamoDBKmsKey.Arn
 
-  RevokedUserCredentialsTable:
```

Comment:
Can we leave this table intact for now, to be deleted as part of https://govukverify.atlassian.net/browse/PYIC-7101

(we want that to be done as a specific task, since it'll delete the VCs in the table permanently)